index.html

```diff
@@ -0,0 +1,183 @@
+            />
+          </li>
+          <li>
+            <img
+              class="social-icon"
+              src="./images/home/twitter.svg"
+              alt="twitter"
+            />
+          </li>
+          <li><a href="#" class="top-header">English</a></li>
+          <li><a href="#" class="top-header">My page</a></li>
+          <li><a href="#" class="top-header">Log out</a></li>
+        </ul>
+      </div>
+      <nav>
+        <div class="logo">
+          <img src="./images/microsoft-logo.svg" alt="logo" />
+        </div>
+        <ul class="parent-link">
+          <li><a href="./index.html">Home</a></li>
+          <li><a href="./about.html">About</a></li>
+          <li>Programe</li>
+          <li>Join</li>
+          <li>Sponser</li>
+          <li>News</li>
+          <li><a class="highlight" href="" id="top-icon">Microsoft</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <!-- hero section -->
+      <section class="hero">
+        <nav class="mobile">
+          <div class="burger">
+            <span></span>
+            <span></span>
+            <span></span>
+          </div>
+          <div class="mobile-menu">
+            <ul class="lin-parent">
+              <li><a href="./index.html">Home</a></li>
+              <li><a href="./about.html">About</a></li>
+              <li>Programe</li>
+              <li>Join</li>
+              <li>Sponser</li>
+              <li>News</li>
+              <li><a class="highlight" href="#" id="top-icon">Microsoft</a></li>
+            </ul>
+            <div class="close">
+              <img src="./images/home/XIcon.svg" alt="close" />
+            </div>
+          </div>
+        </nav>
+        <div class="inner">
+          <span>"Hello! Sharing world"</span>
+          <h1>
+            creative commons <br />
+            global summit <br />
+            2022
+          </h1>
+          <div class="box-desc">
+            <p>
+              A joy celebration beliving in the value of openness and sharing,
+              creating <br />
+              a postive change with people over 80 countries is taking place in
+              October, in korea
+            </p>
+          </div>
+          <div class="info">
+            <p class="history">2022.10.15(THU) ~ 16(FRI)</p>
+            <p class="copy-right">
+              @ National Museum of korea, Art Center Nabi and more
+            </p>
+          </div>
+        </div>
+      </section>
+
+      <!-- main programe section -->
+      <section class="main-programe">
+        <h3 class="sec-title">Main Programe</h3>
+        <article class="cards-group">
+          <div class="card">
+            <img src="./images/home/research.svg" alt="icon" />
+            <h4 class="card-title">Lecture</h4>
+            <p class="card-desc">
+              listen to the speakers from various countries about the message of
+              sharing and opening.
+            </p>
+          </div>
```

Comment:
- [ ]  For semantic purpose, can you please use as article tag to wrap all the cards? The <article> tag specifies independent, self-contained content. An article should make sense on its own.